dev-util/cmark/cmark-0.26.1.recipe

```diff
@@ -0,0 +1,75 @@
+SUMMARY="CommonMark parsing and rendering library and program in C"
+DESCRIPTION="cmark is the C reference implementation of CommonMark, \
+a rationalized version of Markdown syntax with a spec. \
+(For the JavaScript reference implementation, see commonmark.js.) \
+It provides a shared library (libcmark) with functions for parsing \
+CommonMark documents to an abstract syntax tree (AST), manipulating the AST, \
+and rendering the document to HTML, groff man, LaTeX, CommonMark, or an XML \
+representation of the AST. It also provides a command-line program (cmark) \
+for parsing and rendering CommonMark documents."
+HOMEPAGE="https://github.com/jgm/cmark"
+COPYRIGHT="2014 John MacFarlane"
+LICENSE="BSD (2-clause)"
+REVISION="1"
+SOURCE_URI="https://github.com/jgm/cmark/archive/0.26.1.tar.gz"
+CHECKSUM_SHA256="b50615a97f9c19e353d65f3bdbd6898ed1443a6f49e38f0aa888d5b58867f5d6"
+
+ARCHITECTURES="x86 !x86_gcc2 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libcmark$secondaryArchSuffix = $portVersion compat >= 0.2
+	lib:libcmark$secondaryArchSuffix = 0.26.1 compat >= 0.2
+	cmd:cmark$secondaryArchSuffix = 0.26.1 compat >= 0.2
+	cmd:api_test$secondaryArchSuffix = 0.26.1 compat >= 0.2
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	"
+PROVIDES_devel="
+	libcmark${secondaryArchSuffix}_devel = $portVersion compat >= 0.2
+	devel:libcmark$secondaryArchSuffix = 0.2 compat >= 0.2
```

Comment:
I'm not sure but I think **`PROVIDES_devel`** should be:
```
PROVIDES_devel="
	libcmark${secondaryArchSuffix}_devel = $portVersion compat >= 0
	devel:libcmark$secondaryArchSuffix = $portVersion compat >= 0
	"
```
Anyway, «**`devel:libcmark$secondaryArchSuffix = 0.2 compat >= 0.2`**» is wrong because the version of the lib is not «**`0.2`**» but «**`0.26.1`**».